### ISSUE TITLE
Unhelpful error when using CLI and trying to generate sourcemaps

### ISSUE THREAD

**Issue by CrowdHailer** [rollup/rollup#322]
I am trying to use the `rollup.conf.js` file to generate a source map. My config file is.

```js
import typescript from 'rollup-plugin-typescript';

export default {
  entry: 'assets/scripts/boot.ts',
  format: "iife",
  moduleName: "Lob",
  dest: "public/lob.js",
  sourceMap: true,
  // sourceMapFile: 'public/lob.js.map',

  plugins: [
    typescript({
      sourceMap: true
    })
  ]
};
```

The error I get is `Cannot read property 'split' of undefined`
I get the same error if I explicitly define the source map file by removing the comment on the line in the example above.

**Comment by TrySound** [comment]
@CrowdHailer Show full error with stack trace

**Comment by CrowdHailer** [comment]
```
Cannot read property 'split' of undefined
TypeError: Cannot read property 'split' of undefined
    at decode$1$1 (/home/peter/Workshop14/lob/node_modules/rollup/Volumes/Storage/www/ROLLUP/rollup/node_modules/sourcemap-codec/dist/sourcemap-codec.es6.js:102:22)
    at /home/peter/Workshop14/lob/node_modules/rollup/Volumes/Storage/www/ROLLUP/rollup/src/utils/collapseSourcemaps.js:32:11
    at Array.map (native)
    at /home/peter/Workshop14/lob/node_modules/rollup/Volumes/Storage/www/ROLLUP/rollup/src/utils/collapseSourcemaps.js:30:32
    at Array.map (native)
    at collapseSourcemaps (/home/peter/Workshop14/lob/node_modules/rollup/Volumes/Storage/www/ROLLUP/rollup/src/utils/collapseSourcemaps.js:29:25)
    at Bundle.render (/home/peter/Workshop14/lob/node_modules/rollup/Volumes/Storage/www/ROLLUP/rollup/src/Bundle.js:237:42)
    at Object.write (/home/peter/Workshop14/lob/node_modules/rollup/Volumes/Storage/www/ROLLUP/rollup/src/rollup.js:36:32)
    at /home/peter/Workshop14/lob/node_modules/rollup/bin/runRollup.js:110:18
    at tryCatch (/home/peter/Workshop14/lob/node_modules/rollup/Volumes/Storage/www/ROLLUP/rollup/node_modules/es6-promise/lib/es6-promise/-internal.js:182:12)
```

**Comment by TrySound** [comment]
@Victorystick Seems like typescript transpiller generates map string, but rollup uses objects

**Comment by Victorystick** [comment]
I've published a fix for the TypeScript plugin as 0.2.1. Closing this issue.

**Comment by CrowdHailer** [comment]
Thanks. Already updated
